lib/contentTypeParser.js

```diff
@@ -191,6 +195,16 @@ function defaultJsonParser (req, body, done) {
   done(null, json)
 }
 
+function defaultPlainTextParser (req, body, done) {
+  try {
+    var plainText = body
+  } catch (err) {
+    err.statusCode = 400
+    return done(err, undefined)
+  }
```

Comment:
I don't think this could _ever_ throw, so the try-catch should be removed.

Reply:
Should this be `toString` instead?

```js
var plainText = body.toString()
```

Reply:
I thought about using `.toString()` too @cemremengu, but decided to follow the defaultJsonParser function example. However, I could check for null and return a new error based upon `FST_ERR_CTP_EMPTY_JSON_BODY()` or just use `.toString()`?